modules/create.js

```diff
@@ -0,0 +1,76 @@
+import { deleteTodo, updateTodo } from './crud.js';
+import { getDragAfetrElement, selectTodo } from './crud2.js';
+
+const createTodo = (item, todoListss, todoUl) => {
+  const itemContainer = document.createElement('li');
+  itemContainer.className = 'todo';
+  itemContainer.draggable = true;
+
+  itemContainer.addEventListener('dragstart', (e) => {
+    itemContainer.classList.add('draggable');
+    itemContainer.style.opacity = 0.4;
+    e.dataTransfer.effectAllowed = 'move';
+    e.dataTransfer.setData('text/html', e.target.innerHTML);
+  });
+
+  itemContainer.addEventListener('dragend', () => {
+    itemContainer.style.opacity = 1;
+    itemContainer.classList.remove('draggable');
+  });
+
+  todoUl.addEventListener('dragover', (e) => {
+    e.preventDefault();
+    const draggable = document.querySelector('.draggable');
+    const afterElement = getDragAfetrElement(todoUl, e.clientY);
+    if (afterElement === null) {
+      todoUl.appendChild(draggable);
+    } else {
+      todoUl.insertBefore(draggable, afterElement);
+    }
+  });
+
+  const itemForm = document.createElement('form');
+  itemForm.className = 'todo_form';
+  const todoTitle = document.createElement('input');
+  todoTitle.className = 'todo_title';
+  todoTitle.value = item.description;
+  todoTitle.disabled = item.disabled;
+  const checkBox = document.createElement('input');
```

Comment:
- [ ] [optional]: kindly define your variable before other elements. it makes your code more readable 👓

Reply:
Thank you @mohisa302 
I will implement the suggested changes